bazel/README.md

```diff
@@ -87,6 +87,16 @@ for how to update or override dependencies.
     available via a package manager, but may not be available in default repositories depending on
     OS version. To build against `libc++` build with the `--config=libc++` instead of the
```

Comment:
Let's delete the last sentence of this paragraph, as it is a repetition on the information below.

Reply:
sure

Reply:
@harmonyharmo this sentence is still here?

Reply:
oops, fixed now